recipes/vesin/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "vesin" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: python-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: bb325eb9f41e6632600379b1babd26264859d68b2bd1a8bff22ef23efbd8a689
+
+build:
+  skip: true  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ stdlib("c") }}
+    - cmake
+    - make  # [linux]
+  host:
+    - python
```

This is a bit nasty, but may solve your problem by putting everything into build. I may lead to problems with cross-compiling though.
```suggestion
    - python >=3.9
```

---

I'm not sure I understand? What I'm trying to achieve is reduce the number of variants built (i.e. only build for `os x arch` instead of `os x arch x python-version`). You are saying that not having a `python` dependency in `host` would do this?

I think I'll need to cross compile to build packages for linux-arm64 and macos-arm64, so maybe this is something I could experiment with in a branch of the feedstock?

---

Yes, maybe do that and let's start with building out for every Python version. My suggestion should lead to what you want but for cross-compiled versions, it will be more complicated.

We should soon have support for `abi3`-based packages. You could wait for that and use that approach: https://github.com/conda/ceps/pull/86 Eventhough in your case, it isn't `abi3` but a similar approach.

---

Thanks for the link. Yes `abi3` seems like a good idea here, it's not quite the same mechanism but should yield the same results anyway!